### PR TITLE
Add content to recreate the Docker images

### PR DESCRIPTION
**Pull request by wizmer**

### CODE REVIEW

**Comment by wizmer** [comment]
The build is simply failing because it has already been pushed to pypi but it works.

File: Dockerfile_x86_64 [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+# CMake
+############
+
+# Need to build this from source, since the binaries from Kitware don't
+# run with the old version of the libraries in this Centos version
+RUN curl -OL https://cmake.org/files/v3.7/cmake-3.7.2.tar.gz && tar xfz cmake-3.7.2.tar.gz && cd cmake-3.7.2 && ./configure && make -j2 install
+RUN rm -rf cmake-3.7.2.tar.gz cmake-3.7.2
+
+
+############
```
Comment (mgeplf) [inline Dockerfile_x86_64]: is this still needed? Can build_hdf5 be used like on the mac?

Comment (wizmer) [reply Dockerfile_x86_64]: I just tried locally and it did not work

Comment (mgeplf) [reply Dockerfile_x86_64]: how did it fail?

Comment (wizmer) [reply Dockerfile_x86_64]: I did not find hdf5

Comment (mgeplf) [reply Dockerfile_x86_64]: what did you try to get cmake to see it?  How can I try what you tried?

Comment (wizmer) [reply Dockerfile_x86_64]: `docker run -it bluebrain/morphio_wheel:x86_64 /bin/bash` and then I tried to compile morphio.
I did not tried anything specific to make cmake find it. But since it is working as it is right now, and given creating a new image is something extremely rare, I don't wan't to spend too much time on this.

Comment (mgeplf) [reply Dockerfile_x86_64]: if we can outsource having to deal w/ hdf5, it's a win - especially since this is such a gross hack I did a long time ago, that's now being cut and pasted everywhere :(

Comment (wizmer) [reply Dockerfile_x86_64]: Ok, let me try again

Comment (mgeplf) [reply Dockerfile_x86_64]: it's ok, I'll give it a whirl w/ the command you gave above

Comment (mgeplf) [reply Dockerfile_x86_64]: seems to build for me w/ `build_hdf5 1 8`

Comment (wizmer) [reply Dockerfile_x86_64]: Cool, can you commit your working stuff ?
